ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -40,17 +43,20 @@ const AdvancedGasFeeGasLimit = () => {
     setGasLimitInContext(gasLimit);
     const error = validateGasLimit(gasLimit, minimumGasLimitDec);
     setGasLimitError(error);
-  }, [gasLimit, minimumGasLimitDec, setGasLimitInContext]);
+    setErrorValue('gasLimit', error === 'editGasLimitLessThanMinimum');
```

Comment:
> error === 'editGasLimitLessThanMinimum'

Is `editGasLimitGreaterThanMaximum` not a blocking error then?

Reply:
Right `editGasLimitGreaterThanMaximum` should be a blocking error... because that will fail

Reply:
I might be wrong but this reads as only setting a blocking error if error is `editGasLimitLessThanMinimum`?

Reply:
yeah, I think an update is needed here

Reply:
This `setErrorValue` is needed for blocking error only. It updated the value in context which is used to disable save button.

Other error is just displayed in this component only next to input and is not required to be propagated to the context.

Reply:
Probably fn name `setErrorValue` is not very intuitive, its should be something like `setBlockingErrorValue`.

Reply:
I think we're saying that editGasLimitGreaterThanMaximum should be a blocking error

Reply:
I see, I did not knew transaction with gas limit above that can fail.